static/js/stats/stats.js

```diff
@@ -86,18 +87,9 @@ function update_last_full_update(end_times) {
 }
 
 $(() => {
-    $('span[data-toggle="tooltip"]').tooltip({
-        animation: false,
+    tippy(".last_update_tooltip", {
         placement: "top",
-        trigger: "manual",
     });
-    $("#id_last_update_question_sign")
-        .on("mouseenter", () => {
-            $("span.last_update_tooltip").tooltip("show");
-        })
-        .on("mouseleave", () => {
-            $("span.last_update_tooltip").tooltip("hide");
-        });
```

Comment:
Is it intentional that `stats.js` does not import our own `tippyjs.js`?  

(I think this means our size/etc. settings don't run on this page).

Reply:
Added the defaults here.